pkg/networkmanager/interfaces.js

```diff
@@ -585,6 +585,18 @@
             };
         }
 
+        if (settings.wireguard) {
+            result.wireguard = {
+                listen_port: get("wireguard", "listen-port", ""),
+                peers: get("wireguard", "peers", []).map(peer => ({
+                    publicKey: peer['public-key'].v,
+                    endpoint: peer.endpoint?.v ?? "", // enpoint of a peer is optional
+                    allowedIps: peer['allowed-ips'].v
+                })),
+                private_key: get("wireguard", "private-key")
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230703-110332-55dbfd0f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)